docs/localdev.md

```diff
@@ -1,88 +1,139 @@
-2. Clone the repo from your github account to your local machine
-3. Change directory `cd <projects-dir>/external-dns-operator`
-4. Ensure your can access your CRC environment
-   ```bash
-   $ export KUBECONFIG=~/.crc/machines/crc/kubeconfig
-   ```
-5. Access to the CRC internal registry
-   ```bash
-   # assume that podman (tool used to create and maintain containers) is our default
-   # assume our default user is kubeadmin
-   $ podman login -u kubeadmin -p $(oc whoami -t) default-route-openshift-image-registry.apps-crc.testing --tls-verify=false
-   ```
-6. We can now follow the standard make recipes
-   ```bash
-   # create the external-dns-operator namespace
-   $ oc create ns external-dns-operator
-   # set the IMG envar before executing the `make deploy` command
-   # url format is default-route-openshift-image-registry.apps-crc.testing/<namespace>/<image-name>:tag
-   $ export IMG=default-route-openshift-image-registry.apps-crc.testing/external-dns-operator/external-dns-operator:dev
-   $ CONTAINER_ENGINE=podman make image-build
-   $ CONTAINER_ENGINE=popdman TLS_VERIFY=false make image-push
-   # check that the image was pushed
-   $ oc -n external-dns-operator get is
-   ``` 
-7. Now you can deploy an instance of ExternalDNS
-   ```bash
-   # deploy the operator
-   $ make deploy
-   # switch to the external-dns-namespace - so that we don't need to type out -n external-dns-operator all the time
-   $ oc project external-dns-operator
-   # check deployment
-   $ oc get all
-   # if there are errors/problems
-   $ oc get events
-   ```
-8. Run the following command to create the credentials secret for AWS.\
-   Ensure you have your AWS credentials set up in `~/.aws/credentials`.
-   ```bash
-   $ oc create secret generic aws-access-key \
-   --from-literal=aws_access_key_id=${ACCESS_KEY_ID} \
-   --from-literal=aws_secret_access_key=${ACCESS_SECRET_KEY}
-   ```
-   *Note*: other provider options can be found in `api/v1alpha1/externaldns_types.go`, e.g. the `ExternalDNSAWSProviderOptions` structure for AWS.
-
-   Execute the following
-   ```bash
-   $ oc apply -k config/samples/aws
-   # check for the newly created ExternalDNS
-   $ oc get externaldns sample -o yaml  
-   ``` 
-   *Note*: other providers are available in `config/samples/`
-
-### Running the external-dns-operator locally (with CRC)
-
-1. Assume that the operator has been deployed (step 7 in the previous section)
-2. Execute the following
+### Build and push the docker image:
+1. Clone the `external-dns-operator` repository from your GitHub account to your local machine.
+2. Navigate to the downloaded `external-dns-operator` repository:
+    ``` bash
+    $ cd <projects-dir>/external-dns-operator
+    ```
+    `<projects-dir>` is the directory where you downloaded the  `external-dns-operator` repository.
+3. Run `docker login` and provide your login  credentials for quay.io to access the registry.
```

Comment:
No need to login to `quay.io` anymore as the image will be loaded to the node directly.